homework/index.js

```diff
@@ -30,13 +30,118 @@
     return elem;
   }
 
+  function setInnerText(repository, nthChild, text) {
+    document.querySelector('tbody').childNodes[nthChild].lastChild.innerText = text;
```

Comment:
You don't seem to be using `repository` in this function at all. Can you remove it from the parameters list please?

Reply:
I removed the unused parameter. Thanks :pray: